board/power_saving.h

```diff
@@ -4,54 +4,56 @@
 int power_save_status = POWER_SAVE_STATUS_DISABLED;
 
 void power_save_enable(void) {
```

Comment:
These should really be refactored into set_power_save_state(int state)

Reply:
see 96fb376

Reply:
well, let me refactor more. Reading the code, there is a lot of repeated stuff

Reply:
See https://github.com/commaai/panda/pull/226/commits/ec3d0386c2f6e546cc2e5ee8cc855086e8afdbf6